### PR TITLE
Add warning to Recents settings when user doesn't set LC as the Recent provider

### LINKED ISSUE #3884: Warn that Recents cannot be customized without QuickstepSwitcher
### Describe the bug

None of the settings for Recents seem to have any effect. 

### Steps to reproduce

1. Go to Home Settings
2. Select Recents
3. Change any settings 
4. Swipe up to see that none of the changes have applied

### Expected behavior

Settings changed in Recents apply

### Screenshots

https://github.com/LawnchairLauncher/lawnchair/assets/44733677/76ed6cd5-e5f7-41a6-adeb-bb7ed8561a57

### Device information

- Device: Google Pixel 7
- OS: GrapheneOS 14 Beta UQ1A.231205.015.2023121200
- App version: https://github.com/LawnchairLauncher/lawnchair/commit/d0b805d6b9fda7351c3dc291b04479028975e088

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by validcube**
## Description

Clarify why Recent settings do nothing when Lawnchair isn't set as a Recent provider.

* close: #3884

## Type of change

:x: General change (non-breaking change that doesn't fit the below categories like copyediting)
:x: Bug fix (non-breaking change which fixes an issue)
✅ New feature (non-breaking change which adds functionality)
:x: Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: lawnchair/src/app/lawnchair/ui/preferences/destinations/QuickstepPreferences.kt [hunk 1]
```diff
@@ -94,3 +103,16 @@ fun QuickstepPreferences() {
...
+
+@Composable
+fun SettingsIsIgnoreWarning() {
+    Surface(
+        modifier = Modifier.padding(horizontal = 16.dp),
+        shape = MaterialTheme.shapes.large,
+        color = androidx.compose.material3.MaterialTheme.colorScheme.errorContainer,
+    ) {
+        WarningPreference(
+            text = "You are currently using a development build, this settings will be ignored by Lawnchair"
```
Comment (validcube) [inline lawnchair/src/app/lawnchair/ui/preferences/destinations/QuickstepPreferences.kt]: Fix code styling: Spotless

```suggestion
            text = "You are currently using a development build, this settings will be ignored by Lawnchair",
```

File: lawnchair/src/app/lawnchair/ui/preferences/destinations/QuickstepPreferences.kt [hunk 2]
```diff
@@ -94,3 +103,17 @@ fun QuickstepPreferences() {
...
+
+@Composable
+fun QuickSwitchIgnoredWarning() {
+    Surface(
+        modifier = Modifier.padding(horizontal = 16.dp),
+        shape = MaterialTheme.shapes.large,
+        color = androidx.compose.material3.MaterialTheme.colorScheme.errorContainer,
+    ) {
+        WarningPreference(
+
```
Comment (Goooler) [inline lawnchair/src/app/lawnchair/ui/preferences/destinations/QuickstepPreferences.kt]:
```suggestion
```

File: lawnchair/res/values/strings.xml [hunk 3]
```diff
@@ -400,7 +400,9 @@
     <!-- QuickstepPreferences -->
     <!-- <string name="quickstep_label" /> -->
     <!-- <string name="general_label" /> -->
-      <string name="translucent_background">Translucent Background</string>
+    <string name="quickswitch_ignored_warning">Lawnchair isn\'t set as Recent provider, this settings will be ignored by Lawnchair</string>
```
Comment (SuperDragonXD) [inline lawnchair/res/values/strings.xml]:
```suggestion
    <string name="quickswitch_ignored_warning">These settings will be ignored as Lawnchair is not set as Recent provider.</string>
```

File: lawnchair/res/values/strings.xml [hunk 4]
```diff
@@ -400,7 +400,9 @@
     <!-- QuickstepPreferences -->
     <!-- <string name="quickstep_label" /> -->
     <!-- <string name="general_label" /> -->
-      <string name="translucent_background">Translucent Background</string>
+    <string name="quickswitch_ignored_warning">These settings will be ignored as Lawnchair is not set as Recent provider.</string>
```
Comment (suphon-t) [inline lawnchair/res/values/strings.xml]:
```suggestion
    <string name="quickswitch_ignored_warning">These settings will be ignored as Lawnchair is not set as the Recents provider.</string>
```

**Review by suphon-t — APPROVED** [review]

**Review by MrSluffy — APPROVED** [review]

**Comment by ThatOneCalculator** [comment]
I think that it should say "...Recents Provider by QuickstepSwitcher", since many people (including me) didn't know that was a prerequisite.

**Pull request merged**